infra/mesh.tf

```diff
@@ -0,0 +1,20 @@
+module "mesh" {
+  source = "git::https://github.com/nhsdigital/terraform-aws-mesh-client.git//module?ref=v2.1.5"
+
+  name_prefix                    = "local-immunisation"
```

Comment:
We could use this as an opportunity to replace the `local-immunisation` naming with something that makes a bit more sense, e.g. `imms-${var.environment}`